ui/src/app/job-list/table/table.component.html

```diff
@@ -41,23 +41,38 @@
       </button>
       <mat-menu #labelMenu="matMenu" class="details-menu">
         <mat-card>
-          <mat-card-title>{{ j.name }}</mat-card-title>
-          <p>{{ j.status }}</p>
-          <p>Submitted: {{ j.submission | jmShortDateTime }}</p>
+          <mat-card-subtitle>{{ j.status }}</mat-card-subtitle>
+          <mat-card-subtitle>Submitted: {{ j.submission | jmShortDateTime }}</mat-card-subtitle>
+          <mat-divider></mat-divider>
+          <mat-card-actions>
+            <mat-list>
+              <mat-list-item>
+                <a [routerLink]="[j.id]" [queryParams]="getQueryParams()">
+                  <clr-icon shape="arrow" dir="right" size="24"></clr-icon>
+                  <span>Job details</span>
+                </a>
+              </mat-list-item>
+              <mat-list-item *ngIf="canAbort(j)">
+                <a href="#" (click)="abortJob(j)">
+                  <clr-icon shape="times-circle" size="24"></clr-icon>
+                  <span>Abort job</span>
+                </a>
+              </mat-list-item>
+              <mat-list-item *ngIf="hasTimingUrl(j)">
+                <a href="{{ j.extensions.timingUrl }}" target="_blank">
+                  <clr-icon shape="hourglass" size="24"></clr-icon>
+                  <span>Timing Diagram</span>
+                </a>
+              </mat-list-item>
+              <mat-list-item *ngIf="j.name">
+                <a [href]="'?q=name%3D' + j.name">
```

Comment:
I'm not sure this is going to work if you already have filters applied. Also should clarify what the desired behavior would be in that case:

- Should it intersect with existing filters?
- Or, discard existing filters and just apply this

Reply:
Those are good points – I will re-work how this is implemented and let you know when it's ready for re-review.

Reply:
@calbach I've generalized the functionality into something that can be re-used for other column's filters.